civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -4709,11 +4712,42 @@ function civihr_employee_portal_form_user_profile_form_alter(&$form, &$form_stat
         '#markup' => render($yoti_block_view['content']),
       ];
     }
+
+    // To show "Yoti Unlink" on user edit form
+    if ($user_in_form_is_logged_in_user && YotiHelper::getYotiUserProfile($form['#user']->uid)) {
+      $form['yoti_unlink'] = [
+        '#type' => 'item',
+        '#markup' => '<div class="yoti-connect">' .
+        '<a  class="button" onclick="return confirm(\'' .
+        t('This will unlink your account from Yoti.') .
+        '\')" id="yoti-unlink-button" href="' .
+        url('/yoti/unlink') . '">' .
```

Comment:
This depends on internal details of the Yoti module, meaning it can break if it changes. Let's see if there's a better and upgrade-safe solution for this

Reply:
I just talked with @mi-kon about it and said to proceed with this approach.